.github/workflows/test-and-publish.yml

```diff
@@ -27,13 +27,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: docker-compose build
-      - run: docker-compose up --abort-on-container-exit --exit-code-from e2e
+      - run: docker compose build
```

Comment:
might be wise to to switch this to the cypress action so it doesn't have to build docker images.  That would put it in line with RACR and Gameifai clients.

Reply:
What's the reasoning for no longer using docker images?

Reply:
It simplifies the process of running the tests,  We no longer need to maintain an extra docker compose file and a separate docker build file for testing.

Reply:
The question then would be why did we use them in the first place? I thought we wanted to use them to ensure a consistent working environment among other benefits